zerver/tests/test_docs.py

```diff
@@ -106,6 +106,51 @@ def _test(
                     ['<meta name="robots" content="noindex,nofollow" />'], result
                 )
 
+    def test_api_doc_undocumented_endpoint(self) -> None:
+        # nasty mocking to get down into the KeyError
```

Comment:
Can you also add a docstring or something explaining what this test is getting at -- what is the real error condition that it is testing the error handling for?

Reply:
DONE: moved to PR https://github.com/zulip/zulip/pull/22057